datasets/index.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: single
+author_profile: false
+title: Zarr Datasets
+sidebar:
+  title: "Content"
+  nav: sidebar
+---
+
+<font size="4">
+According to the <a href="https://zarr.dev/adopters/">website</a>, Zarr is
+utilised by a variety of research groups, universities, organisations, and
+open-source communities.<br><br>
+
+We have compiled a list of public datasets for the community to use.<br><br>
```

Comment:
I'd avoid "we" here since we're also asking for submissions. Maybe something about this being a sampling of datasets, i.e. not complete or anything.